src/CarbonAware.Aggregators/src/CarbonAware.Aggregators.csproj

```diff
@@ -5,6 +5,7 @@
     <ImplicitUsings>enable</ImplicitUsings>
     <Nullable>enable</Nullable>
     <EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
+    <IsPackable>false</IsPackable>
```

Comment:
Note: we set `IsPackable` to false because we don't want the `dotnet pack` command to pack these individual projects but instead have it referenced through the library